treeherder/model/fixtures/repository.json

```diff
@@ -944,21 +944,6 @@
         "description": ""
     }
 },
-{
-    "pk": 75,
```

Comment:
Deleting this won't delete the repository on production, so it's best to leave this in and use it for one of the new entries below instead.

Reply:
Yeah :-)